lib/core/src/lib/styles/_reference-variables.scss

```diff
@@ -27,3 +27,7 @@ $adf-ref-height: 40px;
 $adf-ref-width: 40px;
 $adf-ref-line-height: 40px;
 $adf-ref-margin-right: 8px;
+$adf-metadata-property-panel-border-color: rgba(0, 0, 0, 0.12);
```

Comment:
Should we really introduce new colors/designs for snackbars instead of using the same which are used across whole application? It will cause inconsistency I guess... @MichalKinas @DenysVuika